src/backoffice/core/components/backoffice-frame/backoffice-header.element.ts

```diff
@@ -1,28 +1,25 @@
 import { UUITextStyles } from '@umbraco-ui/uui-css/lib';
-import { css, CSSResultGroup, html, LitElement, unsafeCSS } from 'lit';
+import { css, CSSResultGroup, html, LitElement } from 'lit';
 import { customElement } from 'lit/decorators.js';
-import logoImg from '/umbraco_logomark_white.svg';
 
 import './backoffice-header-sections.element';
 import './backoffice-header-apps.element';
 
 @customElement('umb-backoffice-header')
 export class UmbBackofficeHeaderElement extends LitElement {
-	
-
 	render() {
 		return html`
 			<div id="appHeader">
 				<uui-button id="logo" look="primary" label="Umbraco" compact>
-					<img src="${logoImg}" alt="Umbraco" />
+					<img src="/umbraco_logomark_white.svg" alt="Umbraco" />
```

Comment:
Did you test this against an Umbraco instance? When running against that, the backoffice is hosted in a directory (/umbraco/) and in my testing, it didn't pick up the configured base URL from vite.cms.config.ts unless you actually import the file first and echo it out hence why images were laid out like this.

Reply:
That's true! We need to do some URL prefixing to get that to work in the "real" CMS. I changed this because it seemed to be some Vite configuration magic, and it breaks when I try to do a "simple" build of the packages as modules. I might be able to fix it with some Rollup configs, but it might also be great to be able to easily prefix paths?

Reply:
I think this is the only way to get Vite to prefix paths at least 🤔 but the backoffice-* elements don't belong in a library do they?

Just come to think of it, we haven't worked with anything regarding loading assets from a package yet. You need a way to access the server URL, which is currently exposed through a context token, but haven't really tested it.